### DISCUSSION TITLE
Hosting Publii from/blog/

### DISCUSSION THREAD

**Discussion by scotdill** [GetPublii/Publii#1458] [Help]
Hi Publii Community.
I have a company website, that I want to keep as is, but I want to then add a subdomain under /blog/ where publii then gets published.
My main site is static HTML so want to host it all on Netlify. 
Any advice or suggestions to make this happen?
I appreciate all responses.

**Comment by 3muchfun** [comment]
Hi, I would suggest to have a look under "Server" on the left hand side. There you can define the path of website under  "Website URL".  @bobmitro, do you agree?
